### DISCUSSION TITLE
Help required for Angular, Nestjs monorepo

### DISCUSSION THREAD

**Discussion by bilalthepunjabi** [vercel/turborepo#1296] [Help]
Is there any example setup guide for two angular projects and one nestjs project in turbo repo?

**Comment by projectashik** [comment]
https://github.com/projectashik/turborepo-nestjs-angular-ts-template

**Reply by peterkracik** [reply]
@projectashik how did you make imported packages work in the Angular app?
I'm trying to do it in my project, I can import it in the code, but then in the browser I get
`typeError: (0 , math_helpers__WEBPACK_IMPORTED_MODULE_0__.add) is not a function` 

```typescript
// packages/math-helpers/src/index.js
export const add = (a: number, b: number) => {
  return a + b;
};
```


(following the [official example](https://turborepo.org/docs/handbook/sharing-code/internal-packages#6-configuring-your-app))

Do I need to configure Angular similar way as it is [required for next.js](https://turborepo.org/docs/handbook/sharing-code/internal-packages#6-configuring-your-app)?
